src/styles/app.css

```
@@ -281,8 +281,8 @@
   --bn-colors-hovered-background: var(--accent);
 
   /* Selected state */
-  --bn-colors-selected-text: var(--primary-foreground);
-  --bn-colors-selected-background: var(--primary);
+  --bn-colors-selected-text: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)